### ISSUE TITLE
Update build and release report scripts

### ISSUE THREAD

**Issue by hesara** [vaadin/framework8-issues#225]
The scripts generating the build and release reports in CI do not exactly match the release process and may have extra or missing parts/links, sections in an inconvenient order etc. The reports from the last build should be compared with the release checklist and the scripts updated to better support our releases.

**Comment by Artur-** [comment]
Need to consider #330 and see if we can remove or simplify something in the process before we start updating scripts

**Comment by hesara** [comment]
Simplifying the reports and making them match the current release process with more automated checks.

**Comment by hesara** [comment]
First steps for the publish report: https://dev.vaadin.com/review/#/c/14732/

**Comment by hesara** [comment]
Steps to do now done, others postponed in separate tickets.
